### PR TITLE
adding section header to .pylintrc

### PR DESCRIPTION
**Pull request by wissamg**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
in the nightly sdk flow we failed to run sdk_pylint_check.sh

### CODE REVIEW

File: Tests/scripts/sdk_pylint_check.sh [hunk 1]
```diff
@@ -5,9 +5,9 @@ echo Checking for sdk related pylint errors
 pylint_tests_result="$(python3 -m pylint --errors-only ./Tests | grep demisto_sdk)"
 pylint_utils_result="$(python3 -m pylint --errors-only ./Utils | grep demisto_sdk)"
 
-if [ -n "$pylint_test_result" ] || [ -n "$pylint_utils_result" ]; then
-    echo Pylint result on Test directory: $pylint_test_result
-    echo Pylint result on Utils directory: $pylint_utils_result
+if [ -n "$pylint_tests_result" ] || [ -n "$pylint_utils_result" ]; then
```
Comment (tkatzir) [inline Tests/scripts/sdk_pylint_check.sh:8]: Why not split it to 2 different `if`s?
What if there are errors only in one of them, why print the other?

**Review by yaakovi — COMMENTED** [review]
I agree with Tal

File: Tests/scripts/sdk_pylint_check.sh (hunk 1 above)
Comment (wissamg) [reply Tests/scripts/sdk_pylint_check.sh:8]: Actually the other should be empty , ":"
But I will  did it in the dummy way.

**Review by yaakovi — APPROVED** [review]

File: Tests/scripts/sdk_pylint_check.sh [hunk 2]
```diff
@@ -5,10 +5,15 @@ echo Checking for sdk related pylint errors
...
+if [ -n "$pylint_tests_result" ]; then
     echo Pylint result on Test directory: "$pylint_tests_result"
+fi
+if [ -n "$pylint_utils_result" ]; then
     echo Pylint result on Utils directory: "$pylint_utils_result"
     exit 1
 fi
+if [ -n "$pylint_tests_result" ]; then
+    exit 1    
+fi
```
Comment (tkatzir) [inline Tests/scripts/sdk_pylint_check.sh:17]: Suggest:

```
if [ -n "$pylint_tests_result" ]; then
    echo Pylint result on Test directory: "$pylint_tests_result"
fi
if [ -n "$pylint_utils_result" ]; then
    echo Pylint result on Utils directory: "$pylint_utils_result"
fi
if [ -n "$pylint_tests_result" ] || [ -n "$pylint_utils_result" ]; then
    exit 1    
fi
```

Comment (tkatzir) [reply Tests/scripts/sdk_pylint_check.sh:17]: Maybe even better to create (boolean) fields, instead of repeating the checks.

**Pull request closed without merging**